### PR TITLE
Add July 2019 Newsletter

### LINKED ISSUE #16: July 2019 Python Pune Newsletter Collection Box
Let's use this issue to collect python related news, new package updates and events happening around Pune. We can compile and publish it during July pythonpune meetup.

## How to add news:
Comment on this issue with news link and news topic and also include the source from we got this.

## How to add new package release:
Paste the link of new package release

## How to include events here:
if a new event is happening in Pune and want to share in the newsletter, feel free to drop the event link with date and venue.

### PR DESCRIPTION
**Pull request by chkumar246**
Fixes #16

Signed-off-by: Chandan Kumar (raukadah) <raukadah@gmail.com>

### CODE REVIEW

**Comment by raukadah** [comment]
The CI is broken check issue #29.

**Comment by digitronik** [comment]
@chkumar246 CI not broken; It was build fails which sometime occur in Travis. I restarted build and it works fine. 

Telegram invite link fails on travis. Which should not fails but its not returning header as `text/html` so `mdcheckr` not able to resolve curl request. I am going to add enhancement of ignore keyword like `#noqa`.

For not ignore this error. I am going to add patch in `mdcheckr` :+1:

File: 2019/July/community_news.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  https://glyph.twistedmatrix.com/2019/06/kernel-python.html
+
+* Pointers in Python: What's the Point? 
+  https://realpython.com/pointers-in-python/
+
+* What’s happening with that pip dependency resolver? 
+  https://pradyunsg.me/blog/2019/06/23/pip-update/
+
+* Building reproducible Python applications for secured environments from rootconf 2019 
+  https://www.youtube.com/watch?v=WAKk83RVtLE&list=PL279M8GbNsesB7sxLO5zNbBHkOVvb7uvy&index=26&t=0s
```
Comment (bhavin192) [inline 2019/July/community_news.md]:
```suggestion
  https://www.youtube.com/watch?v=WAKk83RVtLE
```

File: 2019/July/community_news.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+## Upcoming events in & around Pune
+
+* Rootconf Pune 2019 on 21st Sept, 2019, CFP is open 
+  https://hasgeek.com/rootconf/2019-pune/
+  Discount Code for attendeess: 
+  https://hasgeek.com/rootconf/2019-pune/?code=community10punert.4ybX3EUJRmeu7Di2SUWzIg.WGMtAMlB21UFcei6U2ufrG6RXS0
+
+## Amazing People
+* Nikhil Dhandhre enabled IRC telegram bot for Pythonpune
+  * Here is the pythonpune telegram group:
```
Comment (bhavin192) [inline 2019/July/community_news.md]:
```suggestion
  * Here is the PythonPune telegram group: 
```

File: 2019/July/community_news.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    https://t.me/joinchat/A3Rp5BDl9LpWMHujwZpGAw
+
+* Bhavin, Vishal & Akshay Gaikwad for leading July Python Pune 2019 meetup
+  * https://www.meetup.com/PythonPune/events/262769950/
+
+## Compiled by PythonPune
+   * Akshay Gaikwad
+   * Chandan Kumar
+   * Sourabh Deshmukh
+   * Vishal Vijay Raghavan
```
Comment (bhavin192) [inline 2019/July/community_news.md]:
```suggestion
   * Vishal Vijayraghavan
```

File: 2019/July/community_news.md (hunk 1 above)
Comment (raukadah) [reply 2019/July/community_news.md]: Done

File: 2019/July/community_news.md (hunk 2 above)
Comment (raukadah) [reply 2019/July/community_news.md]: Done

File: 2019/July/community_news.md (hunk 3 above)
Comment (raukadah) [reply 2019/July/community_news.md]: Done

**Comment by raukadah** [comment]
@bhavin192 Updated the pull request.

**Pull request merged**